<ui version="4.0" >
 <class>mainWindow</class>
 <widget class="QMainWindow" name="mainWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>809</width>
    <height>934</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>BrewTarget</string>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>26</y>
     <width>809</width>
     <height>885</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_9" >
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_18" >
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_8" >
        <item>
         <widget class="QLabel" name="label_12" >
          <property name="text" >
           <string>Recipe</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox_recipe" />
        </item>
        <item>
         <spacer name="verticalSpacer_2" >
          <property name="orientation" >
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0" >
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox" >
        <property name="title" >
         <string>Your Recipe</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_12" >
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2" >
           <property name="spacing" >
            <number>1</number>
           </property>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout" >
             <item>
              <widget class="QLabel" name="label" >
               <property name="sizePolicy" >
                <sizepolicy vsizetype="Preferred" hsizetype="Expanding" >
                 <horstretch>20</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text" >
                <string>Name</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit_name" >
               <property name="sizePolicy" >
                <sizepolicy vsizetype="Preferred" hsizetype="Minimum" >
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize" >
                <size>
                 <width>80</width>
                 <height>0</height>
                </size>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2" >
             <item>
              <widget class="QLabel" name="label_2" >
               <property name="sizePolicy" >
                <sizepolicy vsizetype="Preferred" hsizetype="MinimumExpanding" >
                 <horstretch>20</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text" >
                <string>Style</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="comboBox_style" >
               <property name="sizePolicy" >
                <sizepolicy vsizetype="Preferred" hsizetype="Minimum" >
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize" >
                <size>
                 <width>80</width>
                 <height>0</height>
                </size>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3" >
             <item>
              <widget class="QLabel" name="label_3" >
               <property name="sizePolicy" >
                <sizepolicy vsizetype="Preferred" hsizetype="MinimumExpanding" >
                 <horstretch>20</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text" >
                <string>Equipment</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="comboBox_equipment" >
               <property name="sizePolicy" >
                <sizepolicy vsizetype="Preferred" hsizetype="Minimum" >
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize" >
                <size>
                 <width>80</width>
                 <height>0</height>
                </size>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4" >
             <item>
              <widget class="QLabel" name="label_4" >
               <property name="sizePolicy" >
                <sizepolicy vsizetype="Preferred" hsizetype="MinimumExpanding" >
                 <horstretch>20</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text" >
                <string>Batch Size (l)</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit_batchSize" >
               <property name="sizePolicy" >
                <sizepolicy vsizetype="Preferred" hsizetype="Minimum" >
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize" >
                <size>
                 <width>80</width>
                 <height>0</height>
                </size>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_5" >
             <item>
              <widget class="QLabel" name="label_5" >
               <property name="sizePolicy" >
                <sizepolicy vsizetype="Preferred" hsizetype="MinimumExpanding" >
                 <horstretch>20</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text" >
                <string>Boil Size (l)</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit_boilSize" >
               <property name="sizePolicy" >
                <sizepolicy vsizetype="Preferred" hsizetype="Minimum" >
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize" >
                <size>
                 <width>80</width>
                 <height>0</height>
                </size>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_6" >
             <item>
              <widget class="QLabel" name="label_6" >
               <property name="sizePolicy" >
                <sizepolicy vsizetype="Preferred" hsizetype="MinimumExpanding" >
                 <horstretch>20</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text" >
                <string>Efficiency (%)</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit_efficiency" >
               <property name="sizePolicy" >
                <sizepolicy vsizetype="Preferred" hsizetype="Minimum" >
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize" >
                <size>
                 <width>80</width>
                 <height>0</height>
                </size>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout" >
           <property name="spacing" >
            <number>1</number>
           </property>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_7" >
             <item>
              <widget class="QLabel" name="label_7" >
               <property name="sizePolicy" >
                <sizepolicy vsizetype="Preferred" hsizetype="MinimumExpanding" >
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize" >
                <size>
                 <width>20</width>
                 <height>0</height>
                </size>
               </property>
               <property name="text" >
                <string>OG</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLCDNumber" name="lcdNumber_og" >
               <property name="sizePolicy" >
                <sizepolicy vsizetype="Minimum" hsizetype="Minimum" >
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize" >
                <size>
                 <width>80</width>
                 <height>0</height>
                </size>
               </property>
               <property name="segmentStyle" >
                <enum>QLCDNumber::Flat</enum>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_8" >
             <item>
              <widget class="QLabel" name="label_8" >
               <property name="sizePolicy" >
                <sizepolicy vsizetype="Preferred" hsizetype="MinimumExpanding" >
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize" >
                <size>
                 <width>20</width>
                 <height>0</height>
                </size>
               </property>
               <property name="text" >
                <string>FG</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLCDNumber" name="lcdNumber_fg" >
               <property name="sizePolicy" >
                <sizepolicy vsizetype="Minimum" hsizetype="Minimum" >
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize" >
                <size>
                 <width>80</width>
                 <height>0</height>
                </size>
               </property>
               <property name="segmentStyle" >
                <enum>QLCDNumber::Flat</enum>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_9" >
             <item>
              <widget class="QLabel" name="label_9" >
               <property name="sizePolicy" >
                <sizepolicy vsizetype="Preferred" hsizetype="MinimumExpanding" >
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize" >
                <size>
                 <width>20</width>
                 <height>0</height>
                </size>
               </property>
               <property name="text" >
                <string>ABV (%)</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLCDNumber" name="lcdNumber_abv" >
               <property name="sizePolicy" >
                <sizepolicy vsizetype="Minimum" hsizetype="Minimum" >
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize" >
                <size>
                 <width>80</width>
                 <height>0</height>
                </size>
               </property>
               <property name="segmentStyle" >
                <enum>QLCDNumber::Flat</enum>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_10" >
             <item>
              <widget class="QLabel" name="label_10" >
               <property name="sizePolicy" >
                <sizepolicy vsizetype="Preferred" hsizetype="MinimumExpanding" >
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize" >
                <size>
                 <width>20</width>
                 <height>0</height>
                </size>
               </property>
               <property name="text" >
                <string>Bitterness (IBU)</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLCDNumber" name="lcdNumber_ibu" >
               <property name="sizePolicy" >
                <sizepolicy vsizetype="Minimum" hsizetype="Minimum" >
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize" >
                <size>
                 <width>80</width>
                 <height>0</height>
                </size>
               </property>
               <property name="segmentStyle" >
                <enum>QLCDNumber::Flat</enum>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_11" >
             <item>
              <widget class="QLabel" name="label_11" >
               <property name="sizePolicy" >
                <sizepolicy vsizetype="Preferred" hsizetype="MinimumExpanding" >
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize" >
                <size>
                 <width>20</width>
                 <height>0</height>
                </size>
               </property>
               <property name="text" >
                <string>Color (SRM)</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLCDNumber" name="lcdNumber_srm" >
               <property name="sizePolicy" >
                <sizepolicy vsizetype="Minimum" hsizetype="Minimum" >
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize" >
                <size>
                 <width>80</width>
                 <height>0</height>
                </size>
               </property>
               <property name="segmentStyle" >
                <enum>QLCDNumber::Flat</enum>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <spacer name="verticalSpacer" >
             <property name="orientation" >
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0" >
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="horizontalSpacer" >
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0" >
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_2" >
      <property name="title" >
       <string>Recipe Ingredients</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_7" >
       <property name="spacing" >
        <number>1</number>
       </property>
       <property name="leftMargin" >
        <number>5</number>
       </property>
       <property name="topMargin" >
        <number>1</number>
       </property>
       <property name="rightMargin" >
        <number>5</number>
       </property>
       <property name="bottomMargin" >
        <number>5</number>
       </property>
       <item>
        <widget class="QGroupBox" name="groupBox_3" >
         <property name="title" >
          <string>Fermentables</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_13" >
          <item>
           <widget class="FermentableTableWidget" name="fermentableTable" >
            <property name="sizePolicy" >
             <sizepolicy vsizetype="Minimum" hsizetype="Expanding" >
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize" >
             <size>
              <width>0</width>
              <height>20</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_3" >
            <property name="spacing" >
             <number>1</number>
            </property>
            <item>
             <widget class="QPushButton" name="pushButton_addFerm" >
              <property name="text" >
               <string>Add</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_removeFerm" >
              <property name="text" >
               <string>Remove</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_editFerm" >
              <property name="text" >
               <string>Edit</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_4" >
         <property name="title" >
          <string>Hops</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_14" >
          <item>
           <widget class="HopTableWidget" name="hopTable" />
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_4" >
            <property name="spacing" >
             <number>1</number>
            </property>
            <item>
             <widget class="QPushButton" name="pushButton_addHop" >
              <property name="text" >
               <string>Add</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_removeHop" >
              <property name="text" >
               <string>Remove</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_editHop" >
              <property name="text" >
               <string>Edit</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_5" >
         <property name="title" >
          <string>Miscellaneous</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_15" >
          <item>
           <widget class="MiscTableWidget" name="miscTable" />
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_5" >
            <property name="spacing" >
             <number>1</number>
            </property>
            <item>
             <widget class="QPushButton" name="pushButton_addMisc" >
              <property name="text" >
               <string>Add</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_removeMisc" >
              <property name="text" >
               <string>Remove</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_editMisc" >
              <property name="text" >
               <string>Edit</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_6" >
         <property name="title" >
          <string>Yeast</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_16" >
          <item>
           <widget class="YeastTableWidget" name="yeastTable" />
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_6" >
            <property name="spacing" >
             <number>1</number>
            </property>
            <item>
             <widget class="QPushButton" name="pushButton_addYeast" >
              <property name="text" >
               <string>Add</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_removeYeast" >
              <property name="text" >
               <string>Remove</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_editYeast" >
              <property name="text" >
               <string>Edit</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_17" >
      <item>
       <spacer name="horizontalSpacer_2" >
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0" >
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_save" >
        <property name="text" >
         <string>Save</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_clear" >
        <property name="text" >
         <string>Clear</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_exit" >
        <property name="text" >
         <string>Exit</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>809</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile" >
    <property name="title" >
     <string>File</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuAbout" >
    <property name="title" >
     <string>About</string>
    </property>
    <addaction name="actionAbout_BrewTarget" />
   </widget>
   <addaction name="menuFile" />
   <addaction name="menuAbout" />
  </widget>
  <widget class="QStatusBar" name="statusbar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>911</y>
     <width>809</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <action name="actionAbout_BrewTarget" >
   <property name="text" >
    <string>About BrewTarget</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>FermentableTableWidget</class>
   <extends>QTableView</extends>
   <header>FermentableTableWidget.h</header>
  </customwidget>
  <customwidget>
   <class>HopTableWidget</class>
   <extends>QTableView</extends>
   <header>HopTableWidget.h</header>
  </customwidget>
  <customwidget>
   <class>MiscTableWidget</class>
   <extends>QTableView</extends>
   <header>MiscTableWidget.h</header>
  </customwidget>
  <customwidget>
   <class>YeastTableWidget</class>
   <extends>QTableView</extends>
   <header>YeastTableWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
